src/commands.cpp

```diff
@@ -426,3 +439,70 @@ int AOClient::genRand(int min, int max)
     return random_number;
 #endif
 }
+
+void AOClient::diceThrower(int argc, QStringList argv, RollType Type)
+{
+    QString sender_name = ooc_name;
+    int max_roll_faces = server->getDiceValue("max_value");
+    int max_roll_amount = server->getDiceValue("max_dices");
+    int bounded_faces;
+    int bounded_amount;
+    QString dice_results;
+
+    if (argc == 0)
+    {
+        dice_results = QString::number(genRand(1, 6)); // Self-explanatory
+    }
+    else if (argc == 1)
+    {
+        bounded_faces = qBound(1, argv[0].toInt(), max_roll_faces); // faces, max faces
+        dice_results = QString::number(genRand(1, bounded_faces));
+    }
+    else if (argc == 2)
+    {
+        bounded_faces = qBound(1, argv[0].toInt(), max_roll_faces); // 1, faces, max faces
+        bounded_amount = qBound(1, argv[1].toInt(), max_roll_amount); // 1, amount, max amount
+
+        for (int i = 1; i <= bounded_amount ; i++) // Loop as multiple dices are thrown
+        {
+            QString dice_result = QString::number(genRand(1, bounded_faces));
+            if (i == bounded_amount)
+            {
+                dice_results = dice_results.append(dice_result);
+            }
+            else
+            {
+                dice_results = dice_results.append(dice_result + ",");
+            }
+        }
+    }
```

Comment:
This looks like it can get really slow, really fast. Has any testing been done on how this performs?